web_widget_number_ux_choice/static/src/scss/number_ux_choice.scss

```
@@ -0,0 +1,11 @@
+
+.input_number_ux_choice {
+    width: 71% !important;
```

Avoid `!important` always if you can.

```suggestion
    width: 71%;
```

OTOH, to make sane widths and heights, I think it's better to leverage the bootstrap framework, even more in v12 where it's BS4 and uses flexboxes.

---

I can't manage to avoid ! important
And I also try flexboxes or to bootstrap it with container but you need to encapsule it in div and I didn't manage to put my field in div without breaking the widget magic..